src/routes/components/FormItemPage.tsx

```diff
@@ -181,6 +188,112 @@ export default function FormItemPage() {
             </Sandbox>
 
             <ComponentProperties properties={componentProps} />
+
+            <h2 id="component-examples" className="hidden" aria-hidden="true">Examples</h2>
+
+            <h3 id="component-example-sortable-columns">Slotted Helper Text</h3>
+            <Sandbox skipRender >
+              <GoAFormItem label="First name" helpText="dummyValue" >
+                <GoAInput onChange={noop} value="" name="item" />
+                <div slot="helptext">
+                  <span>This is </span>
+                  <span style={{ fontStyle: 'italic' }}>slotted </span>
+                  <span>help text.</span>
+                </div>
+              </GoAFormItem>
+            
+              {language === "react" && (
+                <CodeSnippet
+                  lang="typescript"
+                  tags="react"
+                  allowCopy={true}
+                  code={`
+                  <GoAFormItem label="First name">
+                    <GoAInput onChange={onChange} value={value} name="item"></GoAInput>
+                    
+                    <div slot="helptext">
+                      <span>This is </span>
+                      <span class="italic-helptext">slotted </span>
+                      <span>help text.</span>
+                    </div>
```

Comment:
The react helptext will be passed in via it's props rather than a slot.

Reply:
Done...Code snippets updated